apps/api/src/app/controllers/__tests__/createTestModule.ts

```diff
@@ -134,6 +134,8 @@ import {
     TranslatePlaylistNameCommandHandler,
 } from '../../../domain/models/playlist/commands';
 import {
+    DeleteResource,
```

Comment:
We really need to start looking at how to get away from using this utility for integration tests.